UnitySDK/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -164,7 +157,7 @@ public abstract class Agent : MonoBehaviour
 
         /// Unique identifier each agent receives at initialization. It is used
         /// to separate between different agents in the environment.
-        int m_Id;
+        int m_EpisodeId;
```

Comment:
Sorry, I could have sworn there was an `Id()` method.

Comment on this needs to be updated ("receives at initialization" is wrong now).

Reply:
Done